pyartcd/pyartcd/github.py

```diff
@@ -0,0 +1,166 @@
+import requests
+import base64
+import os
+import ruamel.yaml
+from pyartcd import exceptions
+
+
+class GithubClient:
+    """A GitHub API Client (functions tailored for gen-assembly)"""
+
+    TOKEN = os.environ.get("GITHUB_TOKEN")
+    REPO = "ocp-build-data"
+    OWNER = "openshift-art-build-bot"
+    PATH = "releases.yml"
+    UPSTREAM_REPO = "openshift"
+    GIT_AUTHOR = "AOS Automation Release Team"
+    GIT_EMAIL = "noreply@redhat.com"
+
+    def __init__(self, base, data, assembly_name, temp_branch):
```

Comment:
Understood! Since I extended [ghapi](https://github.com/fastai/ghapi/tree/master/), I need to keep `repo`, `owner` and `token` as class variables. Everything else I've changed to function parameters